src/tenants/planet/LeaderBoard/components/Stories.tsx

```diff
@@ -8,30 +8,40 @@ interface Props {}
 
 export default function Stories({}: Props): ReactElement {
   const { t, ready } = useTranslation(['planet']);
-  return ready ? (
-    <div className={styles.wrapper}>
-      <div className={styles.container}>
-        <div className={styles.storyCard}>
-          <img src={'/tenants/planet/images/leaderboard/restoreTrees.svg'} />
-          <div className={styles.storyContent}>
-            <h2>{t('planet:restoreTrees')}</h2>
-            <div>
-              <img src={'/tenants/planet/images/leaderboard/playIcon.svg'} />
-              <p>{t('planet:watchVideo')}</p>
-            </div>
+  return (
+    <>
+      {ready ? (
+        <div className={styles.wrapper}>
+          <div className={styles.container}>
+            <a className={styles.storyCard} href="https://a.plant-for-the-planet.org/principles">
+              <img
+                src={'/tenants/planet/images/leaderboard/restoreTrees.svg'}
+              />
+              <div className={styles.storyContent}>
+                <h2>{t('planet:restoreTrees')}</h2>
+                <div>
+                  <img
+                    src={'/tenants/planet/images/leaderboard/mapIcon.svg'}
+                  />
+                  <p>{t('planet:learnMore')}</p>
+                </div>
+              </div>
+            </a>
+            <a className={styles.storyCard} href="https://a.plant-for-the-planet.org/standards">
```

Comment:
Same as above, but there is no German page for https://a.plant-for-the-planet.org/de/standards existing yet (but maybe will be added later on).